pollux/lib/anoncreds/src/main/scala/io/iohk/atala/pollux/anoncreds/Models.scala

```diff
@@ -257,6 +258,47 @@ object Credential {
   }
 }
 
+// ****************************************************************************
+case class CredentialAndRequestedAttributesPredicates(
+    credential: Credential,
+    requestedAttribute: Seq[String],
+    requestedPredicate: Seq[String],
+)
+
+object CredentialAndRequestedAttributesPredicates {
+  given Conversion[CredentialAndRequestedAttributesPredicates, UniffiCredentialRequests] with {
```

Comment:
in my personal opinion a good name for `CredentialAndRequestedAttributesPredicates` is `CredentialRequests` or AnonCredsV1.CredentialRequests (if we can isolate the types the corresponds to the V1)